lib/std/zig/BuiltinFn.zig

```diff
@@ -152,6 +152,8 @@ needs_mem_loc: MemLocRequirement = .never,
 eval_to_error: EvalToError = .never,
 /// `true` if the builtin call can be the left-hand side of an expression (assigned to).
 allows_lvalue: bool = false,
+/// `true` if builtin call available outside function scope
+valid_outside_function: bool = true,
```

Comment:
In general, it is better to default to false:

```suggestion
illegal_outside_function: bool = false,
```

Reply:
> In general, it is better to default to false:

+